interface/src/app/mlmodel/mLModelComponents.js

```diff
@@ -0,0 +1,25 @@
+angular.module('adage.mlmodel.components', [
+  'adage.mlmodel.resource'
+])
+
+.component('mLModelSelector', {
+  templateUrl: 'mlmodel/selector.tpl.html',
+  bindings: {
+    selectedMLModel: '='
+  },
+  controller: ['MLModel', '$log', function(MLModel, $log) {
+    var self = this;
+
+    MLModel.get(
```

Comment:
I agree that it's a good idea to use either `{limit: 0}` or implement pagination to avoid future bugs.